pkg/maps/lbmap/ipv4v6_test.go

```diff
@@ -13,10 +14,11 @@ import (
 
 func TestLbAlgAndSessionAffinityTimeout(t *testing.T) {
 	var (
-		alg1  loadbalancer.SVCLoadBalancingAlgorithm = 1
-		alg2  loadbalancer.SVCLoadBalancingAlgorithm = 2
-		time1 uint32                                 = 8
-		time2 uint32                                 = 16
+		alg1        loadbalancer.SVCLoadBalancingAlgorithm = 1
+		alg2        loadbalancer.SVCLoadBalancingAlgorithm = 2
+		time1       uint32                                 = 8
+		time2       uint32                                 = 16
+		invalidTime uint32                                 = uint32(math.Exp2(24))
```

Comment:
nit:  we could just do 1 << 24 instead of casting out of the float exp value.

Reply:
Good idea! I was unsuccessfully looking up an exponentiation operator in Golang, but binary shift also works great here 😊  Done.